tests/device/poly_kybd_test.py

```
@@ -0,0 +1,82 @@
+
+def _make_keeb() -> PolyKybd:
+    keeb = PolyKybd(DeviceSettings(), PolySettings())
+    hid = MagicMock()
+    hid.send_and_read_validate.return_value = (True, _GET_LANG_ACK)
+    keeb.hid = hid
+    return keeb
+
+
+class TestEnumerateLangMultiPacket(unittest.TestCase):
+
+    def _setup_reads(self, keeb: PolyKybd, extra_packets: list[bytes] = None):
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)